packages/app-project/src/screens/SubjectTalkPage/components/SubjectTalkData/components/Discussions/Discussions.jsx

```diff
@@ -12,6 +12,14 @@ import Discussion from '../Discussion'
 import ParticipantsAndComments from '../ParticipantsAndComments'
 import SectionHeading from '../SectionHeading'
 
+const StyledDiscussions = styled(Box)`
+  min-height: auto;
+
+  @media screen and (max-width: 1280px) {
+    max-height: 400px;
+  }
+`
```

Comment:
Something is amiss with the CSS used to constrain the Discussions section. Here's a screenshot of the Comments overflowing into the metadata section and below:
<img width="585" height="778" alt="Screenshot 2025-09-24 at 3 32 44 PM" src="https://github.com/user-attachments/assets/0b734e77-321f-4fd2-9f88-23f445808b3f" />

Reply:
Oops yes that's not right! Fixed in [10da0a3](https://github.com/zooniverse/front-end-monorepo/pull/7046/commits/10da0a354dc47910fb24cdb0bbb223c376814067).
<img width="710" height="650" alt="discussions_wtih_scroll" src="https://github.com/user-attachments/assets/29caf3fd-d0e0-4bf7-b314-d1fb657e84c9" />